src/state.rs

```diff
@@ -95,6 +96,9 @@ pub struct ResolvedDnsStore {
     // in a future with support for per-pod DNS resolv.conf settings we may need
     // to change this to a map from source workload uid to resolved IP addresses.
     by_hostname: HashMap<String, ResolvedDns>,
+
+    // sf_by_hostname is a map which store IP addresses for unkonwn domain after DNS resolving tasks
```

Comment:
nit: the comment is not right

Reply:
Nice catch! It does not store the IP addresses,  done!